e2e/__tests__/toMatchInlineSnapshotWithPretttier3.test.ts

```diff
@@ -66,10 +64,10 @@ test('supports passing `null` as `prettierPath`', () => {
   expect(exitCode).toBe(0);
 });
 
-test('supports passing `prettier-2` as `prettierPath`', () => {
+test('supports passing `prettier` as `prettierPath`', () => {
   writeFiles(DIR, {
     'jest.config.js': `
-        module.exports = {prettierPath: require.resolve('prettier-2')};
+        module.exports = {prettierPath: require.resolve('prettier/index.cjs')};
```

Comment:
why not just `prettier`?

Reply:
This is @fisker 's suggestion, perhaps to avoid unexpected use of Prettier 2?

Reply:
Actually I mean both `index.cjs` and `index.mjs` both should be tested, since user can pass prettier path.

Reply:
We use `require` to load it, so ESM will fail

Reply:
The debug apis was actually missing at first, I added in https://github.com/prettier/prettier/pull/13731

Reply:
> We use `require` to load it, so ESM will fail

Al right, didn't know.